layouts/register/types/by-mail.html

```diff
@@ -25,19 +43,20 @@ <h4 class="usa-graphic-list-heading">Step 3</h4>
 <!-- Download form BEGIN -->
 <div class="usa-grid page-inner mail-download">
   {{ range $key, $form := $.Site.Data.registration_forms }}
-    {{ if gt $form.weight 0 }}
-    <a class="usa-button" href="{{ $.Site.BaseURL }}{{ $form.file_path }}" target="_blank">Download the form (PDF)</a>
+    {{ if eq $key $.Site.Params.language }}
+    <a class="usa-button" href="{{ $.Site.BaseURL }}{{ $form.file_path }}" target="_blank">
+      {{ $translation.register.by_mail.download }}
+    </a>
     {{ end }}
   {{ end }}
-  <p>{{ $.Scratch.Get "alt_forms" }}</p>
-  <p>Download the form in other languages:</p>
+  <p>
+    {{ $translation.register.by_mail.download_others }}
+  </p>
   <ul class="usa-unstyled-list mail-download-list">
-    {{ $.Scratch.Set "alt_forms" 0 }}
-    {{ range $.Site.Data.registration_forms }}
-      {{ if lt .weight 1 }}
-        {{ $.Scratch.Add "alt_forms" 1 }}
+    {{ range $key, $form := $.Site.Data.registration_forms }}
+      {{ if ne $key $.Site.Params.language }}
```

Comment:
The check on `line 46` and `line 57` both look for the `config*.toml` language property to determine which PDF is promoted to the top call-to-action button.

:eyes: https://github.com/18F/vote-gov/pull/70/files#r65714633